samples/BenchmarkDotNet.Samples/BenchmarkDotNet.Samples.csproj

```diff
@@ -23,6 +23,11 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\src\BenchmarkDotNet\BenchmarkDotNet.csproj" />
-    <ProjectReference Include="..\..\src\BenchmarkDotNet.Diagnostics.Windows\BenchmarkDotNet.Diagnostics.Windows.csproj" />
+    <ProjectReference Condition=" '$(TargetFramework)' != 'netstandard2.0' " Include="..\..\src\BenchmarkDotNet.Diagnostics.Windows\BenchmarkDotNet.Diagnostics.Windows.csproj" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <Compile Remove="IntroInliningDiagnoser.cs" />
+    <Compile Remove="IntroNativeMemory.cs" />
+    <Compile Remove="IntroTailcall.cs" />
   </ItemGroup>
 </Project>
```

Comment:
as explained above: if the `Form` project is not really using the `Samples` project, we can remove the dependency and revert the changes  applied to this app